packages/authentication/src/privy/use-privy-authentication-adapter.ts

```
@@ -49,12 +50,26 @@ export function usePrivyAuthenticationAdapter(): AuthHook {
     };
   }, [privyUser, authenticated]);
 
+  const handleExportWallet = React.useCallback(async () => {
+    debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)